.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
```

Comment:
Since travis changes are already in develop, rebase to eliminate any files already merged from the comparison.

Reply:
Hi @sirghiny I successfully rebased to develop.